src/app/resources/browser/styles/highlight.css

```
@@ -0,0 +1,18 @@
+/* Highlight on navigation to an anchor. */
+@keyframes targetNavigatedAnimation {
+  from {
```

Stylelint (17.14.0)

[error] Expected "from" to be "0%" (keyframe-selector-notation)

(keyframe-selector-notation)